pyproject.toml

```diff
@@ -9,44 +9,38 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
 alembic = "==1.13.1"
-amqp = "==5.2.0"
-beautifulsoup4 = "==4.12.3"
 boto3 = "^1.29.6"
 botocore = "^1.32.6"
 cachetools = "==5.3.2"
 celery = {version = "==5.3.6", extras = ["redis"]}
-certifi = ">=2022.12.7"
-cffi = "==1.16.0"
-charset-normalizer = "^3.1.0"
 click = "==8.1.7"
 click-datetime = "==0.2"
-click-didyoumean = "==0.3.0"
-click-plugins = "==1.1.1"
-click-repl = "==0.3.0"
-deprecated = "==1.2.14"
-eventlet = "==0.35.2"
+deptry = "==0.8.0"
+eventlet = "==0.34.3"
```

Comment:
This is downgrading `eventlet` for some reason - it's marked at `0.35.2` above, and I know we recently updated it from Dependabot.